docs/inputs.html

```diff
@@ -0,0 +1,137 @@
+                    class="btn btn-link">Contribute <i class="fab fa-github"></i></a>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <div class="py-1 mobile-menu">
+            Menu <i class="fa-solid fa-angle-up"></i>
+        </div>
+        <section class="doc-container">
+            <aside class="drawer">
+                <h3>Get Started</h3>
+                <ul class="drawer-list">
+                    <li><a href="./get-started.html">Installation</a></li>
+                </ul>
+                <h3>Components</h3>
+                <ul class="drawer-list">
+                    <li><a href="./alerts.html">Alerts</a></li>
+                    <li><a href="./avatars.html">Avatars</a></li>
+                    <li><a href="./badges.html">Badges</a></li>
+                    <li><a href="./buttons.html">Buttons</a></li>
+                    <li><a href="./cards.html">Cards</a></li>
+                    <li><a class="drawer-active" href="./inputs.html">Input</a></li>
+                    <li><a href="./images.html">Images</a></li>
+                    <li><a href="./typography.html">Typography</a></li>
+                </ul>
+            </aside>
+            <div class="doc-content">
+                <h2>Input</h2>
+                <hr class="my-1" />
+                <h2>Introduction</h2>
+                <div class="my-1">
+                    <p>
+                        Inputs allow website users to enter or share their information,
+                        across many usecases.
+                    </p>
+                    <p>
+                        Primarily used for Sign-up/Login Pages, Feedbacks and for general
+                        user inputs.
+                    </p>
+                    <p>By default, input fields take width of 100% .</p>
+                </div>
+
+                <h2 class="py-1">Sign-up/Login Form</h2>
+
+                <div class="my-1">
+                    <p>
+                        Marvel UI offers a classes for input and textarea to verify user
+                        credentials
+                    </p>
+                    <p>
+                        Each field can also have it's own validation built in to meet your
+                        website's criterias.
+                    </p>
+                    <p class="py-1">
+                        Use the <span class="highlight">required</span> class to label to
+                        indicate required input. Don't forget to add required attribute to
+                        your input element as well.
+                    </p>
+                </div>
+
+                <section class="example flex-column gap-1">
+                    <div>
+                        <form class="flex-column align-cntr" action="#">
+                            <div class="input-group">
+                                <label class="required" for="name">Full Name</label>
```

Comment:
You can rather wrap the input inside label element to avoid giving for. It is given in case you put label elsewhere and not with the input element.

Reply:
Thanks for the suggestion but right now going with the outside `label` and fixed it with the suggestion from Hamza.